frontend/demo/src/api/webStore.ts

```diff
@@ -0,0 +1,69 @@
+import { AiryConfig } from "./airyConfig";
+import { getCookie, setCookie } from "./cookie";
+import { User } from "../model/User";
+
+export const storeDomainCookie = (key: string) => (token: string) => {
+  let domain = AiryConfig.TOP_DOMAIN;
+
+  // If we are on development or don't have access to {domain} we set it to the current domain
+  if (AiryConfig.NODE_ENV !== "production" || !document.domain.endsWith("")) {
+    domain = document.domain;
```

Comment:
Since we don't do shared credentials, we can also get rid of this code and just store on the current domain